.github/workflows/type-checker.yml

```diff
@@ -12,19 +11,16 @@ jobs:
 
     steps:
       - name: Checkout code
-        uses: actions/checkout@v2
+        uses: actions/checkout@v4
 
       - name: Setup Python
         uses: actions/setup-python@v4
         with:
-          python-version: '3.10'
+          python-version: "3.10"
 
       - name: Install Requirements
         run: |
-          sudo apt-get update &&
-          pip install poetry &&
-          poetry lock &&
-          poetry install
+          pip install mypy
 
       - name: Run type checks
-        run: poetry run pyright
+        run: mypy src --install-types --non-interactive --ignore-missing-imports --exclude "cli/templates"
```

Comment:
Any way we can setup such flags in a conf file somewhere, like pyproject.toml?

Reply:
Yes of course, this was just the first approach to see if it worked